.drone.yml

```diff
@@ -28,9 +28,6 @@ pipeline:
       GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
     commands:
       - make vic-ui-plugins
-    when:
-      repo: vmware/vic-ui
-      branch: [ master, releases/*, refs/tags/* ]
```

Comment:
@jooskim this line here says that the tests are definitely not being run on PRs (and of course I can look at the build output of a PR build and see that they are not being run) but if I remove it like this then they will be from now on.

Reply:
figured it out, sorry.